src/governance.ts

```diff
@@ -418,23 +418,27 @@ export class GovernanceCanister {
   /**
    * Gets the NeuronID of a newly created neuron.
    */
-  public claimOrRefreshNeuronFromAccount = async (
-    request: ClaimOrRefreshNeuronFromAccount
-  ): Promise<NeuronId | undefined> => {
-    // Note: This is an update call so the certified and uncertified services are identical in this case,
-    // however using the certified service provides protection in case that changes.
-    const response =
-      await this.certifiedService.claim_or_refresh_neuron_from_account({
-        controller: request.controller ? [request.controller] : [],
-        memo: request.memo,
-      });
-
-    const { result } = response;
-    if (result.length && "NeuronId" in result[0]) {
-      return result[0].NeuronId.id;
+  public claimOrRefreshNeuronFromAccount = async ({
+    memo,
+    controller,
+  }: {
+    memo: bigint;
+    controller: Principal;
```

Comment:
the controller used to be optional, no mandatory, on purpose?

Reply:
Let me try without it. The type seems to support it.

```
export interface ClaimOrRefreshNeuronFromAccount {
  controller: [] | [Principal];
  memo: bigint;
}
```

Reply:
it worked, set it as optional!

Reply:
cool